binding/go/rhino.go

```diff
@@ -57,12 +57,23 @@ const (
 )
 
 type RhinoError struct {
-	StatusCode PvStatus
-	Message    string
+	StatusCode 		PvStatus
+	Message    		string
+	MessageStack 	[]string
 }
 
 func (e *RhinoError) Error() string {
-	return fmt.Sprintf("%s: %s", pvStatusToString(e.StatusCode), e.Message)
+	var message strings.Builder
+	message.WriteString(fmt.Sprintf("%s: %s", pvStatusToString(e.StatusCode), e.Message))
+
+	if len(e.MessageStack) > 0 {
+		message.WriteString(":")
+	}
+
+	for i, value := range e.MessageStack {
+		message.WriteString(fmt.Sprintf("\n  [%d] %s", i, value))
+	}
```

Comment:
don't we usually put this in the `if` statement? I know it doesn't change the behaviour, but it does read better